experimental/traffic-portal/src/app/core/statuses/statuses-table/statuses-table.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from '@angular/core';
```

Comment:
This file is missing a license header. See other typescript files for a comment block at the top that you can copy.